torch/csrc/jit/codegen/cuda/python_frontend/examples/python_example.py

```diff
@@ -1,5 +1,5 @@
 import torch
-
+import torch._C._nvfuser as f
```

Comment:
nitpick. Trying to keep things consistent, can we merge this line down as 
`from torch._C._nvfuser import Fusion, FusionDefinition, DataType`

Reply:
fixed